src/backend_manager.cc

```diff
@@ -338,19 +338,31 @@ Status
 TritonBackendManager::CreateBackend(
     const std::string& name, const std::string& dir, const std::string& libpath,
     const triton::common::BackendCmdlineConfig& backend_cmdline_config,
-    std::shared_ptr<TritonBackend>* backend)
+    bool& is_python_backend_based, std::shared_ptr<TritonBackend>* backend)
```

Comment:
Would remove reference to avoid confusion or possible bugs
```suggestion
    bool is_python_backend_based, std::shared_ptr<TritonBackend>* backend)
```